sale_stock_delivery_address/models/sale_order_line.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    dest_address_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Destination Address",
+        help="If set this address will be the delivery address instead of the "
+             "one specified in the Sales Order header.",
+        domain="['|', ('id', '=', order_partner_id), "
```

Comment:
Isn't this too restrictive as it does not allow drop shipping?

Reply:
mmm it is a fair point, but I guess I prefer not to cover that remote case and have a more usable and less-error-prone UX.